src/databricks/labs/ucx/contexts/application.py

```diff
@@ -375,6 +376,10 @@ def path_lookup(self):
         # TODO find a solution to enable a different cwd per job/task (maybe it's not necessary or possible?)
         return PathLookup.from_sys_path(Path.cwd())
 
+    @cached_property
+    def session_state(self):
```

Comment:
current session state has to be created per-job-task in `LinterContext`, not per global context. revert.

Reply:
done